frontend/src/components/workspace/users-settings/user-item/add-role/add-role.tsx

```diff
@@ -0,0 +1,7 @@
+import styles from '../../styles.module.scss';
+
+const AddRole: React.FC = () => {
```

Comment:
I think, "AddTeam" will be better
```suggestion
const AddTeam: React.FC = () => {
```

Reply:
Dunno why i decided to name it AddRole :confused: